scripts/ui.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+helpFunc() {
+	echo ""
+	echo "Usage: $0 -b"
+	echo -e "\t-b Specify the build flag (either ce or ee)"
+	exit 1
+}
+
+buildUi() {
+    # Enter UI directory
+    cd ./web/ui/dashboard
```

Comment:
we need to exit incase this line fails
```suggestion
    cd ./web/ui/dashboard || exit 1
```